### PR TITLE
Guides/use webhooks

### PR DESCRIPTION
**Pull request by TatianaLekhatkova**

### CODE REVIEW

File: docs/guides/set-up-webhooks.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
+# Webhooks for flexible scheduling notifications
+Webhooks are the easiest and fastest solution for quick notification. If something happens, they send the automated message to particular URL. It works for notification and also integration with different platforms. So, how does it work in Vyte?
```
Comment (saintmac) [inline docs/guides/set-up-webhooks.md:2]: "an automated to a particular"

Comment (saintmac) [reply docs/guides/set-up-webhooks.md:2]: "It is useful for notifications and integrating with other platforms"

File: docs/guides/set-up-webhooks.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    "organization": "60af5269459aad001d0cb565",
+    "updatedAt": "2021-05-27T08:03:53.775Z",
+    "createdAt": "2021-05-27T08:03:53.775Z",
+    "__v": 0
+}
+
+```
+
+## Delete the hook
+You can delete particular hook using `/hooks/:hook_id` endpoint and DELETE method.
```
Comment (saintmac) [inline docs/guides/set-up-webhooks.md:139]: If you want Vyte to stop calling your server for this hook, you can delete...

File: docs/guides/set-up-webhooks.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+```shell screen-hidden
+curl --request DELETE 'https://api.vyte.in/v2/hooks/60af526954e6605aae0dad4d' \
+--header 'Authorization: 2lnpjjrurrl49xja5oo0qujtl60embr7zppiphc5fcav4n7ycx' \
+--header 'Content-Type: application/json' \
+
+```
+
+If you used Zapier hooks, the way is a bit different: `POST` method for `/hooks/zapier/delete` endpoint.
```
Comment (saintmac) [inline docs/guides/set-up-webhooks.md:148]: no this endpoint is for Zapier it shouldn't be mentionned in that guide

File: docs/guides/set-up-webhooks.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+
+## Last but not least
+Wanna test hooks? Check out our [Webhook tester](https://github.com/saintmac/webhook-tester) for automated testing.
```
Comment (saintmac) [inline docs/guides/set-up-webhooks.md:151]: thanks for advertising my open source projects but this one is for testing the development of webhooks, not for testing out our webhooks

File: docs/guides/set-up-webhooks.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+    updated
+    cancelled
+    voted
+    new_invitee
+
+* User:
+    joined
+
+* Vyteme:
+    updated
```
Comment (saintmac) [inline docs/guides/set-up-webhooks.md:19]: you should explain what each does (it's probably already in the API ref just reuse it)

File: docs/guides/set-up-webhooks.md [hunk 6]
```diff
@@ -0,0 +1,153 @@
+# Webhooks for flexible scheduling notifications
+Webhooks are the easiest and fastest solution for quick notification. If something happens, they send the automated message to particular URL. It works for notification and also integration with different platforms. So, how does it work in Vyte?
+
+## About our hooks
+The hook has two main properties: `target_url` and `action`. Target URL leads to the recipient application (your website or app), `action` defines the type of event happened. For now Vyte uses several hooks:
```
Comment (saintmac) [inline docs/guides/set-up-webhooks.md:5]: it doesn't "lead to an app", it is the API endpoint that our server should call when the `action` happens on Vyte

Comment (saintmac) [reply docs/guides/set-up-webhooks.md:5]: "Vyte lets you register several webhooks"

**Review by saintmac — CHANGES_REQUESTED** [review]
You should explain a bit better the idea behind webhooks.
I've made a few other comments

File: docs/guides/set-up-webhooks.md (hunk 1 above)
Comment (TatianaLekhatkova) [reply docs/guides/set-up-webhooks.md]: done 04ac6d4c68efa1e05553386bbc2d8e0c28248c37

File: docs/guides/set-up-webhooks.md (hunk 2 above)
Comment (TatianaLekhatkova) [reply docs/guides/set-up-webhooks.md]: done 04ac6d4c68efa1e05553386bbc2d8e0c28248c37

File: docs/guides/set-up-webhooks.md (hunk 3 above)
Comment (TatianaLekhatkova) [reply docs/guides/set-up-webhooks.md]: deleted 04ac6d4c68efa1e05553386bbc2d8e0c28248c37

File: docs/guides/set-up-webhooks.md (hunk 4 above)
Comment (TatianaLekhatkova) [reply docs/guides/set-up-webhooks.md]: deleted 04ac6d4c68efa1e05553386bbc2d8e0c28248c37

File: docs/guides/set-up-webhooks.md (hunk 5 above)
Comment (TatianaLekhatkova) [reply docs/guides/set-up-webhooks.md:19]: added the link 04ac6d4c68efa1e05553386bbc2d8e0c28248c37

File: docs/guides/set-up-webhooks.md (hunk 6 above)
Comment (TatianaLekhatkova) [reply docs/guides/set-up-webhooks.md]: done 04ac6d4c68efa1e05553386bbc2d8e0c28248c37

**Review by saintmac — APPROVED** [review]

**Pull request merged**